### PR TITLE
Add Stack outputs to verbose info and deploy output

### PR DESCRIPTION
**Pull request by svdgraaf**
## What did you implement:

***Implementing Issue:*** #2135 

## How did you implement it:

Changed the info command a bit, now it forwards the full `gatherData` hash instead of just the info part to `display()`. I also needed to update all the tests.

## How can we verify it:

As discussed in https://github.com/serverless/serverless/issues/2135, this is a PR to add `--verbose` output to the `info` and `deploy` commands.

When adding the `--verbose` command, this will append your Stack Outputs, like so:

screenshot 2016-10-03 12 52 18

## Todos:

- [x] Write tests
- [x] Write documentation
- [x] Fix linting errors
- [x] Make sure code coverage hasn't dropped
- [x] Provide verification config/commands/resources
- [x] Change ready for review message below

***Is this ready for review?:*** YES

### CODE REVIEW

**Comment by flomotlik** [comment]
GTM from my side on functionality, @pmuens can you review and if the Code is OK and it works as expected after your review you can merge.

File: lib/plugins/aws/info/index.js [hunk 1]
```diff
@@ -88,13 +88,14 @@ class AwsInfo {
       return BbPromise.resolve(gatheredData);
     })
     .then((gatheredData) => this.getApiKeyValues(gatheredData))
-    .then((gatheredData) => BbPromise.resolve(gatheredData.info)) // resolve the info at the end
+    .then((gatheredData) => BbPromise.resolve(gatheredData)) // resolve the info at the end
```
Comment (pmuens) [inline lib/plugins/aws/info/index.js]: Could you update or remove the misleading comment.

**Comment by pmuens** [comment]
@svdgraaf thank you for this PR! Really helpful.

- Could you please add the `verbose` option to the core `info` plugin? (like it's done in the `deploy` plugin)
- Could you please add the `verbose` option at the top of the `docs/03-cli-reference/05-info.md` file? (like it's done in the `deploy` readme file)
- And rebase the branch so that the conflicts are resolved

Thank you very much.

Once those are done it's good to go from my side.

**Comment by svdgraaf** [comment]
@pmuens I think I covered all your requested changes, can you check again?

**Comment by pmuens** [comment]
Awesome! Thank you very much @svdgraaf 🌮 

Merged. Great functionality!

**Pull request merged**